docs/usage/reserved-ip.md

```diff
@@ -130,25 +127,31 @@ reservedip-deploy-6cf9858886-lb7cr   0/1     ContainerCreating   0          35s
 reservedip-deploy-6cf9858886-pkcfl   1/1     Running             0          35s   172.18.42.50   spider-worker
 ```
 
-But when we remove this SpiderReservedIP, everything will return to normal.
+But when we delete this ReservedIP, everything will return to normal.
 
 ```bash
-kubectl delete -f https://github.com/spidernet-io/spiderpool/tree/main/docs/example/reserved-ip/test-ipv4-reservedip.yaml
+kubectl delete -f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/test-ipv4-reservedip.yaml
 ```
 
-Another interesting question is that what happens if an IP address to be reserved has been allocated before SpiderReservedIP is created? Of course, we dare not stop this running Pod and recycle its IP addresses, but SpiderReservedIP will still ensure that no Pod can continue to use this IP address when this Pod releases these IP addresses. Therefore, SpiderReservedIPs should be confirmed as early as possible before network planning, rather than being supplemented at the end of all work.
+Another interesting question is that what happens if an IP address to be reserved has been allocated before ReservedIP is created? Of course, we dare not stop this running Pod and recycle its IP addresses, but ReservedIP will still ensure that when the Pod is terminated, no other Pods can continue to use the reserved IP address.
+
+>Therefore, ReservedIPs should be confirmed as early as possible before network planning, rather than being supplemented at the end of all work.
 
 ### Clean up
 
-Clean the relevant resources so that you can run this tutorial again.
+Clean the relevant resources so that we can run this tutorial again.
 
 ```bash
-kubectl delete -f https://github.com/spidernet-io/spiderpool/tree/main/docs/example/reserved-ip --ignore-not-found=true
+kubectl delete \
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/test-ipv4-reservedip.yaml \
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/test-ipv4-ippool.yaml \
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/reservedip-deploy.yaml \
+--ignore-not-found=true
 ```
 
 ## A Trap
 
-So, can you use SpiderIPPool's field `excludeIPs` to achieve the same effect as SpiderReservedIP? The answer is **NO**! Look at such a case, now you want to reserve an IP `172.18.43.31` for an external application of the cluster, which may be a Redis node. To achieve this, you created such an SpiderIPPool:
+So, can we use IPPool's field `excludeIPs` to achieve the same effect as ReservedIP? The answer is **NO**! Look at such a case, now we want to reserve an IP `172.18.43.31` for an external application of the cluster, which may be a Redis node. To achieve this, we should create such an IPPool:
```

Comment:
`we` is not suggested in a user manual.

Reply:
Fine, I will modify it, but I don't want to replace "IPPool" with "SpiderIPPool".

Reply:
😅  This clause is fine to me. We can modify others later.